website/content/docs/operations/stateful-workloads.mdx

```diff
@@ -14,59 +14,78 @@ This document addresses traditional storage you can attach as a filesystem and d
 
 The [Stateful Workloads with CSI tutorial](/nomad/tutorials/stateful-workloads/stateful-workloads-csi-volumes) and the [Nomad CSI demo repository](https://github.com/hashicorp/nomad/tree/main/demo/csi) offer guidance and examples on how to use CSI plugins with Nomad and include job files for running the plugins and configuration files for creating and consuming volumes.
 
 ### Host volumes
 
-Host volumes mount paths from the host (the Nomad client) into allocations. Nomad is aware host volume availability and uses it for job scheduling. But, Nomad does not know about the volume's underlying characteristics, such as if it is a standard folder on a local ext4 filesystem, backed by a distributed networked storage such as GlusterFS, or a mounted NFS/CIFS volume from a NAS or a public cloud service such as AWS EFS. Because you need to declare host volumes in the agent configuration file, you have to restart the Nomad client to reconfigure them. This makes host volumes impractical if you need to change your storage configuration frequently. 
+Host volumes mount paths from the host (the Nomad client) into allocations. Nomad is aware of host volume availability and makes use of it for job scheduling. However, Nomad does not know about the volume's underlying characteristics, such as if it is a standard folder on a local ext4 filesystem, backed by a distributed networked storage such as GlusterFS, or a mounted NFS/CIFS volume from a NAS or a public cloud service such as AWS EFS. Therefore they can be used both for local somewhat persistent storage and for highly persistent networked storage.
 
-As administrators usually handle storage, operating system (OS), and job configurations separately, make sure that appropriate access controls grant permission to modify and update these configurations. If there is a complete separation of duties for storage, OS, and job management, the different teams must communicate well to mitigate potential issues around storage performance, size limitations, and inappropriate mounting options.
+Because you need to declare host volumes in the Nomad agent's configuration file, you have to restart the Nomad client to reconfigure them. This makes host volumes impractical if you need to change your storage configuration frequently. Furthermore, it might require coordination between different [personnas](/nomad/docs/concepts/security#personas) to configure and consume host volumes, for instance a Nomad Administrator to modify Nomad's configuration file to add/update/remove host volumes to make them available for consumption by Nomad Operators. Also, in the case of networked host volumes, a Storage Administrator might need to provision the volumes and make them available to the Nomad clients, and a System Administrator might need to mount them on the Nomad clients.
```

Comment:
```suggestion
Because you need to declare host volumes in the Nomad agent's configuration file, you have to restart the Nomad client to reconfigure them. This makes host volumes impractical if you need to change your storage configuration frequently. Furthermore, it might require coordination between different [personas](/nomad/docs/concepts/security#personas) to configure and consume host volumes, for instance a Nomad Administrator to modify Nomad's configuration file to add/update/remove host volumes to make them available for consumption by Nomad Operators. Also, in the case of networked host volumes, a Storage Administrator might need to provision the volumes and make them available to the Nomad clients, and a System Administrator might need to mount them on the Nomad clients.
```